cylc/flow/graph_parser.py

```diff
@@ -685,15 +685,17 @@ def _families_all_to_all(
         n_info = []
         n_expr = expr
         for name, offset, trig, opt in info:
-            fam_member = False
             if (name, trig) in family_trig_map:
-                fam_member = True
+                fam = True
```

Comment:
Too short to be self documenting: Consider `is_family` or `family`

Reply:
Good suggestion, but I actually just removed the variable entirely as it was only being used for documentation purposes (not good use of a variable!)